MapboxNavigation/NavigationMapView.swift

```diff
@@ -678,7 +678,7 @@ open class NavigationMapView: MGLMapView, UIGestureRecognizerDelegate {
         let fractionTraveled = routeProgress.fractionTraveled
         
         // In case if route was fully travelled - remove main route and its casing.
-        if fractionTraveled == 1.0 {
+        if Int(fractionTraveled) == 1 {
```

Comment:
Was the bug that `fractionTraveled` could increase beyond 1.0 and get missed by this logic? If so, `>=` would be a little clearer, but this works.

Reply:
Yep, there are cases sometimes when `fractionTraveled` can be `1.0000000000000002` or similar. I guess you're right `>=` will be clearer, will change to use this check.